style.css

```diff
@@ -1,3 +1,315 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+
 body {
-    background-color: lightgray;
+    text-align: center;
+}
+
+
+/* Feel free to remove these styles or customise in your own stylesheet 👍  */
+.attribution {
+    font-size: 11px;
+    text-align: center;
+}
+
+.attribution a {
+    color: hsl(228, 45%, 44%);
+}
+
+.backgroundColor {
+    color: rgb (246, 246, 246);
+    width: 0%;
+
+}
+
+
+.mainContainer {
+    display: flex;
+    text-align: center;
+    width: 100%;
+    /* border: 1px;
+    border-style: solid; */
+}
+
+.mainHeading {
+    font-size: 40px;
+    text-align: left;
+}
+
+.mainText {
+    font-size: 16px;
+    color: darkgray;
+    padding-top: 50px;
+    width: 400px;
+    text-align: left;
+    padding-bottom: 50px;
+}
+
+.mainButton {
+    width: 110px;
+    border-radius: 5px;
+    padding: 5px 8px;
+    border: 1px solid black;
+}
+
+.mainButton a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+.leftColumn {
+    width: 50%;
+    padding-top: 120px;
+    padding-left: 100px;
+
+}
+
+.rightColumn {
+    width: 50%;
+    padding-top: 50px;
+
+}
+
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+* {
```

Comment:
There is some duplication of css rules. 
You will get more maintainable, easy to read & modify CSS if you define rules for one element at a time starting with global styles first (e.g. for `*`, `body`, `h1`, `p`, `a`, `img`), then defining the more specific classes (`leftColumn`, `rightColumn` etc).

Reply:
Hi, christiana
thank you so much for reviewing my code . I will make all the changes and update you ASP.
BR